extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/multiplepersistenceunits/MultiplePersistenceUnitsImportSqlHotReloadScriptTest.java

```diff
@@ -34,14 +32,14 @@ public void testImportSqlScriptHotReload() {
         String expectedName = "import.sql load script entity";
         assertBodyIs(expectedName);
 
-        String hotReloadExpectedName = "modified import.sql load script entity";
-        TEST.modifyResourceFile("import.sql", new Function<String, String>() {
-            @Override
-            public String apply(String s) {
-                return s.replace(expectedName, hotReloadExpectedName);
-            }
-        });
-        assertBodyIs(hotReloadExpectedName);
+        //        String hotReloadExpectedName = "modified import.sql load script entity";
+        //        TEST.modifyResourceFile("import.sql", new Function<String, String>() {
+        //            @Override
+        //            public String apply(String s) {
+        //                return s.replace(expectedName, hotReloadExpectedName);
+        //            }
+        //        });
+        //        assertBodyIs(hotReloadExpectedName);
```

Comment:
Better remove this code instead of commenting out

Reply:
Good point! Fixed